src/rlx_prv_relup.erl

```diff
@@ -133,14 +133,14 @@ make_upfrom_script(State, Release, UpFrom) ->
                              systools:make_relup(CurrentRel, [UpFromRel], [UpFromRel], CorrectOptions)
                      end) of
         ok ->
-            ec_cmd_log:error(rlx_state:log(State),
+            ec_cmd_log:info(rlx_state:log(State),
                           "relup from ~s to ~s successfully created!",
                           [UpFromRel, CurrentRel]),
             {ok, State};
         error ->
             ?RLX_ERROR({relup_script_generation_error, CurrentRel, UpFromRel});
         {ok, RelUp, _, []} ->
-            ec_cmd_log:error(rlx_state:log(State),
+            ec_cmd_log:info(rlx_state:log(State),
```

Comment:
Maybe put this after `write_relup_file\3` below, in case that blows up, even though you already wrote to log it was successful?